app/src/main/scala/org/alephium/explorer/api/EndpointExamples.scala

```diff
@@ -172,6 +172,38 @@ object EndpointExamples extends EndpointsExamples {
       totalCountPerChain = ArraySeq(perChainCount, perChainCount)
     )
 
+  private val explorerInfo =
+    ExplorerInfo(
+      releaseVersion = "1.11.2+17-00593e8e-SNAPSHOT",
+      commit         = "00593e8e8c718d6bd27fe218e7aa438ef56611cc"
+    )
+
+  private val tokenSupply =
+    TokenSupply(
+      timestamp   = ts,
+      total       = U256.MaxValue,
```

Comment:
using MaxValue is a bit too much, let's maybe divide by 10^x to reach a level of magnitude like in https://mainnet-backend.alephium.org/infos/supply

Reply:
I don't how to divide an Integer with a variable function. 10^x is a function. What is value of `x` here?

Please explain.

Reply:
something like: 
```
ALPH.MaxALPHValue.divUnsafe(U256.Billion) = 1000000000000000000
```
so x is 9 in that case: 1 billion = 10^9

the idea is just to make the example more friendly to read, with `U256.MaxValue` it's a bit scary: `115792089237316195423570985008687907853269984665640564039457584007913129639935` :sweat_smile:

Reply:
Should `circulating` should be changed as well? `U256.HalfMaxValue` also looks a bit scary 😅 

```scala
circulating = ALPH.MaxALPHValue.divUnsafe(U256.Billion).divUnsafe(U256.Two)
```

```json
"total"      : "1000000000000000000",
"circulating": "500000000000000000"
```